src/widgets/numeric-refinement-list/numeric-refinement-list.js

```diff
@@ -0,0 +1,193 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
```

Comment:
This widget shouldn't be here, it belongs to the `develop` branch and has been renamed to `numeric-menu` (the file should be deleted).